packages/@okta/vuepress-site/docs/guides/add-an-external-idp/create-an-app-at-idp/oktatookta/appatidp.md

```diff
@@ -1,29 +1,22 @@
-> **Note**: This guide will refer to the Okta org that represents the Identity Provider as **Org A** and the second Okta org as **Org B**.
+When you are configuring federation between two Okta orgs using OpenID Connect:
 
-1. At **Org A**, select **Applications** and then click **Add Application**.
+1. At the Okta org that represents the Identity Provider, select **Applications** and then click **Add Application**.
 
 2. Click **Create New App**.
 
-3. You need a trusted client, so select **Web** as the platform. 
+3. You need a trusted client, so select Web as the platform. OpenID Connect is the sign-in method by default.
```

Comment:
2. You need a trusted client, so select **Web** as the platform. OpenID Connect is the sign-in method by default.